source/java/product-guide/index.markdown

```diff
@@ -1312,6 +1312,16 @@ At the time you create the request, it is likely that you may know the account s
 	AccountStore accountStore = anAccountStoreMapping.getAccountStore();
 	UsernamePasswordRequest authenticationRequest = new UsernamePasswordRequest("usernameOrEmail", "password", accountStore);
     AuthenticationResult result = application.authenticateAccount(authenticationRequest);
+    
+#### Authentication Expansion 
+
+If you want the actual account object returned from a successful authentication attempt, then you can [expand the account in-line](#links-expansion) using `AuthenticationOptions`.  For example:
```

Comment:
Please see my comments in the other PR about adding an Options method argument.  Let me know if there is any confusion.